.github/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM adoptopenjdk/openjdk8 as build
+# use local mvn repo to speedup build
+COPY .m2 /root/.m2
```

Comment:
would we need to add `.m2` to the `.gitignore`?

Reply:
shouldn't be necessary as it's deleted afterwards.